pkg/db-repo/repository.go

```diff
@@ -24,6 +26,7 @@ const (
 
 var operations = []string{Create, Read, Update, Delete, Query}
 var RecordNotFound = gorm.ErrRecordNotFound
+var DuplicateEntry = errors.New("duplicate entry")
```

Comment:
You don't need that anymore

Reply:
actually we also don't need record not found here. Or do we want to keep it for the future?

Reply:
We need RecordNotFound because gorm can return it to us and we then return that to our caller, but our caller should not need to import gorm to check for that error, but instead use db_repo.RecordNotFound.